### PR TITLE
 Adding py-linux-ports recipe

### PR DESCRIPTION
**Pull request by sujitmandal**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sujitmandal** [comment]
@conda-forge-coordinator @conda-forge-admin @conda-forge/help-python The recipe has passed all checks and is ready for your review, Thanks.

File: recipes/py-linux-ports/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set name = "py-linux-ports" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```
Comment (BastianZim) [inline recipes/py-linux-ports/meta.yaml:6]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}
```

File: recipes/py-linux-ports/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (BastianZim) [inline recipes/py-linux-ports/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/py-linux-ports/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  sha256: 2d7e6ecfe4458c9c087d414013ba2961a6a0541c5d441eea8d432d456a31ce01
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (BastianZim) [inline recipes/py-linux-ports/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/py-linux-ports/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```
Comment (BastianZim) [inline recipes/py-linux-ports/meta.yaml:20]:
```suggestion
    - python >=2.7
```

Ref https://github.com/sujitmandal/py-linux-ports/blob/ca08fa238cc40ed0e808f2ee1f02be0d10404a7c/setup.py#L25

But you'll need to confirm if this is correct.

File: recipes/py-linux-ports/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  run:
+    - python >=3.6
```
Comment (BastianZim) [inline recipes/py-linux-ports/meta.yaml:22]:
```suggestion
    - python >=2.7
```

File: recipes/py-linux-ports/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+test:
+  imports:
+    - PyLinuxPorts
```
Comment (BastianZim) [inline recipes/py-linux-ports/meta.yaml:26]:
```suggestion
test:
  imports:
    - PyLinuxPorts
  commands:
    - pip check
  requires:
    - pip
```
This just double checks that the correct versions are installed.

**Comment by sujitmandal** [comment]
@conda-forge-coordinator @conda-forge-admin  The recipe has passed all checks.

**Review by BastianZim — CHANGES_REQUESTED** [review]
Please implement all of my requests before pinging again. It seems like some are still missing.

**Comment by sujitmandal** [comment]
> changes

i have changes all your suggest please check it.

File: recipes/py-linux-ports/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```
Comment (BastianZim) [inline recipes/py-linux-ports/meta.yaml:20]:
```suggestion
    - python >=2.7
```

File: recipes/py-linux-ports/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2d7e6ecfe4458c9c087d414013ba2961a6a0541c5d441eea8d432d456a31ce01
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (BastianZim) [inline recipes/py-linux-ports/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/py-linux-ports/meta.yaml (hunk 1 above)
Comment (BastianZim) [reply recipes/py-linux-ports/meta.yaml:6]: This has not been added yet.

**Review by BastianZim — CHANGES_REQUESTED** [review]

**Review by BastianZim — APPROVED** [review]

**Pull request merged**